docs/v2-migration-guide.rst

```diff
@@ -179,6 +179,78 @@ bug fixes will be shipped to the 1.26.x release stream**.
+  tool like pyenv.
+- **Red Hat Enterpritse Linux 7 (RHEL 7)**: Upgrade to RHEL 8 or RHEL 9.
+- **Read the Docs**: Upgrade your `configuration file to use Ubuntu 22.04
+  <https://docs.readthedocs.io/en/stable/config-file/v2.html>`_. Feel free to use the `urllib3
+  configuration <https://github.com/urllib3/urllib3/blob/2.0.0/.readthedocs.yml>`_ as an
+  inspiration.
+
+docker.errors.dockerexception: error while fetching server api version: request() got an unexpected keyword argument 'chunked'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+docker-py swaps some of the urllib3 internals in an unsupported way. The following pull request
+solves the issue: https://github.com/docker/docker-py/pull/3116. Until it gets reviewed, merged and
+released, your only option is to pin urllib3 to 1.26.x using ``urllib3<2.0``.
+
+ImportError: cannot import name 'gaecontrib' from 'requests_toolbelt._compat'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To be compatible with urllib3 2.0, Requests Toolbelt released version 1.0.0 without Google App
+Engine Standard Python 2.7 support. Most users that reported this issue were using the Pyrebase
+library that provides an API for the Firebase API. This library is unmaintained, see
+https://github.com/thisbejim/Pyrebase/issues/435 for possible replacements.
+
+``ImportError: cannot import name 'DEFAULT_CIPHERS' from 'urllib3.util.ssl_'``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This likely happens because you're using botocore which does not support urllib3 2.0 yet:
+https://github.com/boto/botocore/issues/2921. The good news is that botocore explcitly declares in
+its dependencies that it only supports urllib3<1.27. Make sure to use a recent pip. That way, pip
+will install urllib3 1.26.x until botocore starts supporting urllib3 2.0.
+
+AttributeError: module 'urllib3.connectionpool' has no attribute 'VerifiedHTTPSConnection'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``VerifiedHTTPSConnection`` class has always been documented to be in the
+:mod:`~urllib3.connection` module. It used to be possible to import it from
+:mod:`~urllib3.connectionpool` but that was acccidental and is no longer possible due to a
+refactoring in urllib3 2.0.
+
+Note that the new name of this class is :class:`~urllib3.connection.HTTPSConnection`. It can be used
+starting from urllib3 1.25.9.
+
+AttributeError: 'HTTPResponse' object has no attribute 'strict'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strict parameter is unneeded with Python 3 and should be removed. We are considering adding it
+back to help users of the unmaintained cachecontrol library, see https://github.com/urllib3/urllib3/issues/3010.
```

Comment:
Let's remove the note about adding it back since it'll get removed again regardless of us adding it back or not, instead we can link to the issue and if folks are interested they can read more.

In order for people to support both we can provide a code sample which does `getattr(resp, "strict", True)` noting that default behavior is now `strict=True`.

Reply:
I removed it in [`c9839ed` (#3018)](https://github.com/urllib3/urllib3/pull/3018/commits/c9839ed4ab4d3aace2e16d19acb2e9ebab5db446).

There's no point to use `getattr` here: this parameter is no longer needed with Python 3. So if you're in the position of changing your code, you can just remove strict.